src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Nadahar Thank you very much for the testings :+1: 

Sorry, this time i didn't know it will not succeed, i'm  in unknown territory, but that's already improve, isn't it ? :wink: 
OK, i still to work on it, but i feel that i can make it work.